helpers/3p/utils/lib/createFrame.js

```diff
@@ -4,9 +4,17 @@
  * https://github.com/jonschlinkert/define-property/blob/master/index.js
  */
 
+function extend(obj /* , ...source */) {
```

Comment:
not sure why this fn was in `utils`, the only place it appears to be used is `createFrame`. So moved it here to eliminate the circular dependency.